.flake8

```diff
@@ -1,6 +1,6 @@
 [flake8]
 max-line-length=120
-exclude = .git,__pycache__,dist,node_modules/*,utils/*,tests/functional/test_env/*,sandbox/node_modules/*
+exclude = .git,__pycache__,dist,node_modules/*,utils/*,tests/functional/test_env/*,sandbox/node_modules/*,.venv/*
```

Comment:
I assume this was t fix that issue with flatten?

Am I right in thinking none of our code goes in to the .venv folder? It's just there for packages we pull in?

Reply:
Someone merged the `sandbox/node_modules/*` before me, which resolved the flatten flake issues.

`.venv/*` has appeared in my repo and was causing thousands of lines to be output. It doesn't sit in the repo, so no, none of our code goes in to there.

Reply:
hence I have added it to the end. Not sure why I may be the only one being affected.